.changeset/swift-coats-teach.md

```diff
@@ -0,0 +1,5 @@
+---
+"astro": patch
+---
+
+The devtool bar doesn't report `div` and `span` when check their role. `div` and `span` are special elements that don't have an interaction assigned, instead it is assigned based on the role assigned via attributes.
```

Comment:
It's unclear to me what the PR does from the changeset only. Is that the toolbar **won't** report from now on?

Reply:
Yeah exactly. Let me reword the changeset by adding some examples, then please review it and suggest possible changes.

Reply:
Done in [`782940e` (#10719)](https://github.com/withastro/astro/pull/10719/commits/782940eb29eedac822bc8857bd5ad3fab65198ac)